CHANGELOG.md

```diff
@@ -9,6 +9,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfluff/sqlfluff/releases) and will be copied in here on each release (please remember to update the issues and contributors to links!). There is no need to manually edit this file going forward.
 -->
 
+## [0.8.0] - 2021-11-06
+
+## What’s Changed
+
+This minor release brings an improvement to the performance of the parser, and a rebuild of the Jinja Templater. Lots of dialect improvements have also been done. Full list of changes below:
```

Comment:
```suggestion
This release brings an improvement to the performance of the parser, and a rebuild of the Jinja Templater. Lots of dialect improvements have also been done. Full list of changes below:
```

I'd avoid use of the word minor since < 1.0.0 the mid-point "minor" versions are used for majors too.